functions/openhab.js

```diff
@@ -347,6 +347,8 @@ function syncAndDiscoverDevices(token, success, failure) {
 
 					case 'Lighting':
 						deviceTypes = ['action.devices.types.LIGHT'];
+						traits = getSwitchableTraits(item);
```

Comment:
Have you tested this? Your changes should do absolutely the same thing as it was before, since without "break" it will just continue to execute next line and will call "traits = getSwitchableTraits(item);"